src/bitcoin-tx.cpp

```diff
@@ -358,6 +358,18 @@ vector<unsigned char> ParseHexUO(map<string,UniValue>& o, string strKey)
     return ParseHexUV(o[strKey], strKey);
 }
 
+static CAmount AmountFromValue(const UniValue& value)
```

Comment:
nit: I think we should place this in a utility class and use the same implementation also for bitcoin-txs (`static CAmount AmountFromValue(const UniValue& value)`). Can be done post this PR.